src/train.jl

```diff
@@ -10,6 +10,10 @@ using Zygote: Zygote
 
 export setup, train!
 
+using ProgressLogging: @progress, @withprogress, @logprogress
+using Zygote: Zygote, Params
```

Comment:
```suggestion
using Zygote: Zygote
```
I removed any Params use from the codebase